tests/unit/models/package-info-cache-test.js

```diff
@@ -17,107 +19,121 @@ describe('models/package-info-cache.js', function() {
     cli = new MockCLI({ ui });
   });
 
-  describe('packageInfo contents tests on valid project', function() {
-    beforeEach(function() {
-      projectPath = path.resolve(addonFixturePath, 'simple');
-      packageJsonPath = path.join(projectPath, 'package.json');
-      packageContents = require(packageJsonPath);
-      project = new Project(projectPath, packageContents, ui, cli);
-      pic = project.packageInfoCache;
+  describe('packageInfo', function() {
+    describe('valid project', function() {
+      let project, fixturifyProject;
+      before(function() {
```

Comment:
should this simply be unified with the instantiation tests?

Reply:
I would say no, for the same reason as the comment below.